ci/cron/wednesday.yml

```diff
@@ -16,54 +16,97 @@ jobs:
 - job: open_release_pr
   timeoutInMinutes: 60
   pool:
-    name: linux-pool
-    demands: assignment -equals default
+    vmImage: ubuntu-latest
   steps:
   - checkout: self
     persistCredentials: true
-  - bash: ci/dev-env-install.sh
   - bash: |
-      set -euo pipefail
-      eval "$(./dev-env/bin/dade-assist)"
+      set -euxo pipefail
 
-      setvar() {
-          echo "Setting '$1' to '$2'"
-          echo "##vso[task.setvariable variable=$1;isOutput=true]$2"
+      if header=$(git config 'http.https://github.com/digital-asset/daml.extraheader'); then
+          AUTH="$header"
+      else
+          AUTH="Authorization: basic $(git config remote.origin.url | grep -o '://.*:.*@' | cut -c4- | rev | cut -c2- | rev)"
+      fi
+
+      BASE_SHA=$(git rev-parse HEAD)
+      az extension add --name azure-devops
+      echo "$(System.AccessToken)" | az devops login --org "https://dev.azure.com/digitalasset"
+
+      reset() {
+          git checkout -f $BASE_SHA
+          git reset --hard
       }
+      open_pr() {
+          local branch title body out pr_number
+          branch=$1
+          title="$2"
+          body="$3"
+          out=$4
+          git branch -D $branch || true
+          git checkout -b $branch
+          git add .
+          git -c user.name="Azure Pipelines DAML Build" \
+              -c user.email="support@digitalasset.com" \
+              commit \
+              -m "$(printf "$title\n\n$body\n\nCHANGELOG_BEGIN\nCHANGELOG_END\n")"
+          git push origin $branch:$branch
+          pr_number=$(curl -H "Content-Type: application/json" \
+                           -H "$AUTH" \
+                           --silent \
+                           --location \
+                           -d "{\"title\": \"$title\", \"head\": \"$branch\", \"base\": \"master\", \"body\": \"$body\"}" \
```

Comment:
If you're wary about special characters in the title or body, you could do something like this:

```sh
jq -n --arg branch "$branch" --arg title "$title" --arg body "$body" '{"title": $title, "head": $branch, "base": "master", "body": $body}' \
  | curl ... -d @- https://...
```

Reply:
I did not know `jq` had such a syntax. Thanks!